src/Components/HorizontalRule.js

```diff
@@ -10,7 +10,11 @@ type Props = {
 };
 
 const HorizontalRule = ({ color, thickness }: $Component & Props) => (
-  <Hr color={color || "primary"} thickness={thickness || 2} />
+  <Hr
+    color={color || "primary"}
+    thickness={parseInt(thickness) || 2}
+    style={{ margin: "0 auto" }}
```

Comment:
Would you mind dropping an issue/PR on elevate-ui to update the `Hr` component to use "gutterTop" and "gutterBottom" (or whatever props we are using in the Typography component you set up?